projects/admin/src/app/circulation/checkin/checkin.component.ts

```diff
@@ -300,4 +302,13 @@ export class CheckinComponent implements OnInit {
       );
     }
   }
+
+  /** Reset search input */
+  private _resetSearchInput(): void {
+    setTimeout(() => {
+      this.searchInputDisabled = false;
+      this.searchInputFocus = true;
+      this.searchText = '';
+    }, 300);
```

Comment:
AS you mentioned, you need to be outside of the initial cycle to do that. But a `setTimeout` with a `delay=0` isn't suffisent ? @jma play more than me with this notation (for editor) and could have a better idea.

Reply:
done